portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,88 @@
+                    </svg>
+                </button>
+                <div class="col-8">
+                    <div class="row">
+                        <div class="col-12 col-lg-6 py-lg-5 py-2 px-5">
+                            <div class="d-flex align-items-center bg-white rounded-3 w-100 h-100">
+                                <div class="container my-3">
+                                    <h4 id="projectTitle">Interns GDL Website</h4>
+                                    <p id="projectDescription">A website built with React.js that uses the Blogger Rest API v3 as a Content Manager System.</p>
+                                    <a id="projectRepository" class="btn btn-info m-1" href="https://github.com/interns-gdl/Interns-GDL-Website" target="_blank">See the code</a>
+                                    <a id="projectLink" class="btn btn-info m-1" href="https://interns-gdl.web.app/" target="_blank">View project</a>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="col-12 col-lg-6 py-lg-5 py-2 px-5">
+                            <img id="projectImage" class="img-fluid rounded-3"  src="https://1.bp.blogspot.com/-bsCfc6qwAiA/YAWTsa7W6aI/AAAAAAAAhaQ/7w3Cy4q0desSkg3wrYgtbhP8k9dFHRFugCLcBGAsYHQ/w400-h400/INTERNS%2BGDL.png">
+                        </div>
+                    </div>        
+                </div>
+                <button id="next" class="btn btn-light h-100 col-2" onclick="nextProject()">
+                    <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-caret-right-fill" viewBox="0 0 16 16">
+                        <path d="M12.14 8.753l-5.482 4.796c-.646.566-1.658.106-1.658-.753V3.204a1 1 0 0 1 1.659-.753l5.48 4.796a1 1 0 0 1 0 1.506z"/>
+                    </svg>
+                </button>
+            </div>
+            
+        </div>
+        <hr>
+        <div class="container-fluid">
+            <h1>Experience</h1>
+            <ul>
+                <li>hellou</li>
+            </ul>
+        </div>
+        <hr>
+        <div class="container-fluid">
+            <h1>About me</h1>
+            <ul>
+                <li>Speak spanish, english and a bit of german, hallo.</li>
+                <li>I love fiction, superheroes, harry potter, starwars, etc.</li>
+                <li>My favorite sport is Formula 1.</li>
+                <li>I am a competitive programmer! </li>
+                <li>My favorite artist is Logic and my favorite band is Muse </li>
+            </ul>
+        </div>
+    </main>
+    <footer></footer>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.bundle.min.js" integrity="sha384-b5kHyXgcpbZJO/tY9Ul7kGkf1S0CWuKcCD38l8YkeH8z8QjE0GmW1gYU5S9FOnJ0" crossorigin="anonymous"></script>
+    <script src="script.js"></script>
```

Comment:
FYI, I just looked up where to put script tags and found this interesting. No need to change anything.
https://stackoverflow.com/questions/436411/where-should-i-put-script-tags-in-html-markup